site/content/docs/5.1/components/orange-footer.md

```diff
@@ -7,31 +7,455 @@ aliases: "/docs/5.1/components/orange-footer/"
 toc: true
 ---
 
-## Overview
+## How it works
 
-{{< callout warning >}}
-### Orange footer is not a component yet
+Orange footer is basically a `<footer>` tag with a `.bg-dark` class. It can be composed of several parts:
+- mailing list bar
+- social media bar
+- navigation bar
+- service bar
+- **mandatory** bar
 
-The feature will be delivered with [#892]({{< param repo >}}/issues/892).
+## Examples
 
-In the meantime, some examples have been developed in the [footers examples section]({{< docsref "/examples/footers" >}}) by using mostly Boosted utilities and our Collapse JavaScript plugin.
+<footer role="contentinfo" class="bg-dark">
+  <h2 class="visually-hidden">Site map & informations</h2>
+  <!-- Mailing list -->
+  <div class="container footer-mailing">
+    <h3>Sign up to our mailing list</h3>
+    <form>
+      <label for="inputEmail2" class="visually-hidden">Email</label>
+      <input type="email" class="form-control bg-dark border-dark text-white" id="inputEmail2" placeholder="Enter your email">
+      <button type="submit" class="btn btn-secondary btn-inverse text-nowrap">Sign up</button>
+    </form>
+  </div>
 
-Total responsiveness is not implemented and design guidelines are not perfectly respected but it can help you while waiting for the component's delivery.
-{{< /callout >}}
+  <div class="border-bottom border-1 border-dark"></div>
+
+  <!-- Follow us bar -->
+  <div class="container footer-social">
+    <h3>Follow us</h3>
+    <ul>
+      <li><a href="#" class="btn btn-icon btn-social btn-twitter btn-inverse"><span class="visually-hidden">Twitter</span></a></li>
+      <li><a href="#" class="btn btn-icon btn-social btn-facebook btn-inverse"><span class="visually-hidden">Facebook</span></a></li>
+      <li><a href="#" class="btn btn-icon btn-social btn-instagram btn-inverse"><span class="visually-hidden">Instagram</span></a></li>
+      <li><a href="#" class="btn btn-icon btn-social btn-whatsapp btn-inverse"><span class="visually-hidden">WhatsApp</span></a></li>
+      <li><a href="#" class="btn btn-icon btn-social btn-linkedin btn-inverse"><span class="visually-hidden">LinkedIn</span></a></li>
+      <li><a href="#" class="btn btn-icon btn-social btn-youtube btn-inverse"><span class="visually-hidden">YouTube</span></a></li>
+      <li><a href="#" class="btn btn-icon btn-social btn-snapchat btn-inverse"><span class="visually-hidden">Snapchat</span></a></li>
+      <li><a href="#" class="btn btn-icon btn-social btn-pinterest btn-inverse"><span class="visually-hidden">Pinterest</span></a></li>
+      <li><a href="#" class="btn btn-icon btn-social btn-mail btn-inverse"><span class="visually-hidden">Mail</span></a></li>
+    </ul>
+  </div>
+
+  <div class="border-bottom border-1 border-dark"></div>
+
+  <!-- Navigation -->
+  <div class="navbar-dark footer-nav">
+    <div class="container d-none d-md-flex">
```

Comment:
That was the purpose of megamenu in boosted v4 https://boosted.orange.com/docs/4.6/components/orange-megamenu/ to avoid duplicating content between desktop and mobile view and still switching from flat view to accordion one. The mega-menu haven't been moved to v5, but maybe some solution could be used here?